docs/topic-guides/hp-tuning-det/hp-grid.txt

```diff
@@ -4,7 +4,7 @@ Hyperparameter Search: Grid
 ===========================
 
 The ``grid`` search method generates trials on a "grid" of hyperparameter
-configurations and trains each trial for ``max_steps`` steps. The user specifies
+configurations and trains each trial for ``max_length``. The user specifies
```

Comment:
Seems like we need a unit here (`max_length` records, although I realize it might not be records).